### PR TITLE
update test files before separating them

### PR DESCRIPTION
**Pull request by wilsonwang371**
## Why are these changes needed?

This is a patch that separate compatibility test into different files so that later we can only enable the necessary tests and delay other tests to nightly. 

## Related issue number

#514 #298

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [X] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by kevin85421 — APPROVED** [review]
LGTM. I will rebase #589 after this PR be merged.

File: tests/compatibility-test.py [hunk 1]
```diff
@@ -1,150 +1,19 @@
...
 import unittest
-import subprocess
-from string import Template
-
 import docker
-import time
 
-ray_version = '1.9.0'
-ray_image = "rayproject/ray:1.9.0"
+from kuberay_utils.utils import *
```
Comment (DmitriGekhtman) [inline tests/compatibility-test.py:6]: \* imports are considered bad style for various reasons

Would you mind either
- Importing each utility used
OR
- Doing from `kuberay_utils import utils` and prefixing each utility with `utils.`

**Review by scarlet25151 — APPROVED** [review]
The patch move some function to the utils, overall LGTM.

**Review by DmitriGekhtman — APPROVED** [review]
lgtm!

**Pull request merged**